### PR TITLE
Added pysmbc recipe

### PR DESCRIPTION
**Pull request by arkilic**

### CODE REVIEW

**Comment by arkilic** [comment]
Cousin PR to https://github.com/NSLS-II/lightsource2-recipes/pull/74

File: recipes-tag/pysmbc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  name: pysmbc
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/hamano/pysmbc.git
+  git_rev: {{ version }}
+
+build:
+  number: 1
+  script: python setup.py install
```
Comment (danielballan) [inline recipes-tag/pysmbc/meta.yaml]: We might as well follow conda-forge's pattern and use `python setup.py install --single-version-externally-managed --record=record.txt`

File: recipes-tag/xattr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  name: xattr
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/xattr/xattr.git
+  git_rev: {{ version }}
+
+build:
+  number: 1
+  script: python setup.py install
```
Comment (danielballan) [inline recipes-tag/xattr/meta.yaml]: ...and the same here.

**Comment by danielballan** [comment]
Thanks for the fast turn-around @arkilic.

**Pull request merged**